guides/source/active_record_validations.md

```diff
@@ -278,39 +310,149 @@ end
+validation rules. Each time a validation fails, an error message is added to the
+object's `errors` collection, and this error is associated with the specific
+attribute being validated.
+
+When a validation fails, the error message is stored in the `errors` collection
+under the attribute name that triggered the validation. This means you can
+easily access the errors related to any specific attribute. For instance, if you
+validate the `:name` attribute and the validation fails, you will find the error
+message under `errors[:name]`.
+
+In modern Rails applications, the more concise validate syntax is commonly used,
+for example:
+
+```ruby
+validate :name, presence: true
+```
+
+However, older versions of Rails used "helper" methods, such as:
 
-Active Record offers many pre-defined validation helpers that you can use
-directly inside your class definitions. These helpers provide common validation
-rules. Every time a validation fails, an error is added to the object's `errors`
-collection, and this is associated with the attribute being validated.
+```ruby
+validates_presence_of :name
+```
+
+Both notations perform the same function, but the newer form is recommended for
+its readability and alignment with Rails' conventions.
 
-Each helper accepts an arbitrary number of attribute names, so with a single
-line of code you can add the same kind of validation to several attributes.
+Each validation accepts an arbitrary number of attribute names, allowing you to
+apply the same type of validation to multiple attributes in a single line of
+code.
 
-All of them accept the `:on` and `:message` options, which define when the
-validation should be run and what message should be added to the `errors`
-collection if it fails, respectively. The `:on` option takes one of the values
-`:create` or `:update`. There is a default error message for each one of the
-validation helpers. These messages are used when the `:message` option isn't
-specified. Let's take a look at each one of the available helpers.
+Additionally, all validations accept the `:on` and `:message` options. The `:on`
+option specifies when the validation should be triggered, with possible values
+being `:create` or `:update`. The `:message` option allows you to define a
+custom error message that will be added to the errors collection if the
+validation fails. If you do not specify a message, Rails will use a default
+error message for that validation.
 
 INFO: To see a list of the available default helpers, take a look at
-[`ActiveModel::Validations::HelperMethods`][].
+[`ActiveModel::Validations::HelperMethods`][]. This API section uses the older
+notation as described above.
+
+[`ActiveModel::Validations::HelperMethods`]:
+    https://api.rubyonrails.org/classes/ActiveModel/Validations/HelperMethods.html
+
+Below we outline the most commonly used validations.
+
+### `absence`
+
+This helper validates that the specified attributes are absent. It uses the
```

Comment:
I suppose that the following should be described first for newbies:

```diff
- This helper validates that the specified attributes are absent.
+ This helper validates that the specified attributes or an association are absent.
```

I suppose that the following (passing an association and attributes simultaneously) should also be clarified for newbies:

```ruby
class Supplier < ApplicationRecord
  has_one :account
  validates :account, :name, :login, :email, absence: true # possible? impossible? unrecommended?
end
```